src/GraphQL.Tests/Bugs/Bug956.cs

```diff
@@ -25,9 +25,6 @@ public Bug956QueryType()
                 string asString = ctx.GetArgument<string>("base64");
                 asString.ShouldBe("R3JhcGhRTCE="); // GraphQL!
 
-                var asList = ctx.GetArgument<List<byte>>("base64");
-                asList.Count.ShouldBe(12);
```

Comment:
This test made no sense.  Converting to `List<byte>` produced 12 bytes (conversion of the `string` to `char[]` then to `List<int>`), but converting to `byte[]` produced 8 bytes (decoding the `string` from a base64 string).

As such, there is no practical change here; converting a string to a `byte[]` is assumed to be a base64-decoding operation and still functions.